source/query-with-natural-language/prompt-natural-language-aggregation.txt

```diff
@@ -71,7 +71,12 @@ Steps
 
             Aggregation pipelines can contain operators such as 
             :pipeline:`$merge` and :pipeline:`$out` that can 
-            modify your collection's data.
+            modify your collection's data. 
+
+            If the aggregation pipeline generated by natural language 
+            querying contains these operators, you must confirm 
+            execution of the pipeline on the confirmation modal before 
+            the pipeline is executed.
```

Comment:
(minimalism)
This is pretty subjective. But you might tweak a smidge to use a few less words. Spitballing, but something like:
If an AI-generated aggregation pipeline contains these operators, you are prompted for confirmation before the pipeline is deployed.